### DISCUSSION TITLE
Set `smart_gaps on` as default

### DISCUSSION THREAD

**Discussion by FilippoBonazziSUSE** [openSUSE/openSUSEway#144] [Polls]
We have a suggestion in #141 to add `smart_gaps on` as a default. From `man 5 sway`:

```
smart_gaps on|off|toggle|inverse_outer
           If smart_gaps are on gaps will only be enabled if a workspace has more than one child.
           If smart_gaps are inverse_outer outer gaps will only be enabled if a workspace has exactly one child.
```

I do not use gaps at all, so I have no opinion on this.

**Comment by denisok** [comment]
-1 
I tried it `swaymsg smart_gaps toggle` and for me personally aesthetic of a gap even on single window is better.

this setting could be easily enabled in personal config `~/.config`...

**Comment by FilippoBonazziSUSE** [comment]
Agreed, best left to the user config
